reactApp/src/data/putRating.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+const putRating = async (term, remedy, formValues) => {
+  try {
+    const response = await axios.put(
+      `http://localhost:7000//remedies/${term}`,
```

Comment:
should be `http://localhost:7000/remedies/${term}`, (it has two // in the middle) 
strange enough it does not replace the id at the moment: 
 http://localhost:7000//remedies/[object%20Object]
 If I hardcode the id, like this for example: http://localhost:7000/remedies/641f60b3e7523a06fc0bfa06 - it works, I get a 200!:) 
However, the data that is sent atm in the body needs to be adjusted I think (see below)

Reply:
Good catch, I make the change